src/includes/modal/inputcardterms.pug

```diff
@@ -0,0 +1,16 @@
+.modalwindow(id="modal-inputcardterms")
+    .d-flex
+        h4.modalwindow__title Informativa sul trattamento dei dati personali
+        svg.icon.ml-auto.modalwindow__close.cursor-pointer
+            use(href="/assets/icons/sprite.svg#it-close")
+    h6.modalwindow__desc Ai sensi degli artt.13-14 del Regolamento UE 2016/679
+    .modalwindow__content.mb-3 
```

Comment:
after the merge of this [PR](https://github.com/pagopa/io-pay/pull/25) we have call  to get this info dynamically `/resources`.

> NOTA : this call return 3DS2 configuration setting too. Could be stored it in to `sessionStorage` ? I think could be useful to choice from `/pay` or `/pay3ds2` action